tests/integration/server.int.js

```diff
@@ -1,6 +1,6 @@
 import { mockConfig } from '../mocks';
 import start from '../../src/server';
-import * as appRouteHandler from '../../src/routes/appRouteHandler';
+import * as appRouteHandler from '../../src/plugins/app-route/handler';
```

Comment:
Another thing probably outside the scope of this...

Might be nice to have an integration test for the language redirect behavior.